manifests/params.pp

```diff
@@ -1,7 +1,7 @@
 # Params class for parent foreman_scap_client
 # @api private
 class foreman_scap_client::params {
-  $downcased_fqdn = downcase($facts['fqdn'])
+  $downcased_fqdn = downcase($trusted['certname'])
```

Comment:
Is it needed to downcase the cert name? I'm wondering if the certname is always the same as the filename.

Reply:
Gah it didn't post my reply, sorry Puppetserver forces all lower case it appears so yes, not needed and it's been removed